services/librariesio/librariesio-dependent-repos.tester.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+const Joi = require('joi')
+const ServiceTester = require('../service-tester')
+const { isMetric } = require('../test-validators')
+
+const t = new ServiceTester({
+  id: 'librariesio-dependent-repos',
+  title: 'Libraries.io dependent repos',
+  pathPrefix: '/librariesio/dependent-repos',
+})
+module.exports = t
+
+t.create('dependent repo count')
+  .get('/npm/got.json')
+  .expectJSONTypes(
+    Joi.object().keys({
+      name: 'dependent repos',
+      value: isMetric,
+    })
+  )
+
+t.create('dependent repo count (not a package)')
+  .get('/npm/foobar-is-not-package.json')
+  .expectJSON({
+    name: 'dependent repos',
+    value: 'package not found',
+  })
```

Comment:
I know we've moved the 'unparsable json response' up a level. I don't think we've done that yet with the "inaccessible" test but we probably should. Its nice to have less boilerplate in these test modules.

Reply:
Ignore this - I can see you've already done it in #1890